oneandone/src/main/java/org/apache/jclouds/oneandone/rest/handlers/OneAndOneRateLimitRetryHandler.java

```diff
@@ -41,6 +41,6 @@
    }
 
    public static long millisUntilNextAvailableRequest(long secondsToNextAvailableRequest) {
-      return secondsToNextAvailableRequest * 1000;
+      return secondsToNextAvailableRequest * 8000;
```

Comment:
This should return the seconds until the next available request. The previous value looked OK, as it configures the exact value returned by the provider in the response headers.